Woof/Woof/ViewModel/SitterProfileViewModel.swift

```diff
@@ -0,0 +1,80 @@
+import Foundation
+
+/// The view model for a pet sitter profile view that is responsible for preparing and providing data for it.
+final class SitterProfileViewModel: ObservableObject {
+    // MARK: - Internal interface
+
+    /// The name of the pet sitter.
+    @Published var name: String = ""
+
+    /// The surname of the pet sitter.
+    @Published var surname: String = ""
+
+    /// The phone of the pet sitter.
+    @Published var phone: String = ""
+
+    /// The URL of the pet sitter's avatar image.
+    @Published var avatarUrl: URL?
+
```

Comment:
Do we need this property for now as we do not present an avatar in our views?

Reply:
For now not. The property is removed.
Thanks!